h5netcdf/core.py

```diff
@@ -150,6 +150,15 @@ def __array__(self, *args, **kwargs):
         return self._h5ds.__array__(*args, **kwargs)
 
     def __getitem__(self, key):
+        if h5py.__version__ >= LooseVersion("3.0.0") and self._root.decode_strings:
```

Comment:
This code can probably just check `decode_strings`?
```suggestion
        if self._root.decode_strings:
```

Reply:
I've applied your suggestion using `getattr` since `decode_strings` isn't available in all cases.